content/docs/command-reference/plots/index.md

```diff
@@ -98,72 +100,50 @@ names in the `train` array below:
+create visualizations out of users data. Pre-defined specifications are called
+templates.
 
-Built-in _plot templates_ are stored in the `.dvc/plots/` directory. The default
-one is called `default.json`. It can be changed with the `--template` (`-t`)
-option of `dvc plots show` and `dvc plots diff`. For templates in the
-`.dvc/plots/` directory, the path and the json extension are not required: you
-can specify only the base name e.g. `--template scatter`.
+DVC has a few built-in templates. They are stored within
+[`dvc-render`](https://github.com/iterative/dvc-render) library. The default one
+is called `linear`. It can be changed with the `--template` (`-t`) option of
+`dvc plots show` and `dvc plots diff`. The argument provided to `--template`
+can be either a built-in template name or path to a JSON specification.
+For templates stored in the `.dvc/plots/` directory, the path and the json
+extension are not required: user can specify only the base name e.g.
+`--template scatter`.
 
 DVC has the following built-in plot templates:
 
-- `default` - linear plot
+- `linear` - linear plot, used by default
 - `scatter` - scatter plot
 - `smooth` - linear plot with LOESS smoothing, see
   [example](/doc/command-reference/plots#example-smooth-plot)
 - `confusion` - confusion matrix, see
   [example](/doc/command-reference/plots#example-confusion-matrix)
+- `confusion_normalized` - confusion matrix with values normalized to <0, 1>
+  range
+- `simple` - simplest linear template, good for learning how to work with
+  [`templates`](/doc/command-reference/plots/templates) command
 
-### Custom templates
-
-Plot template files are [Vega-Lite](https://vega.github.io/vega-lite/) files
-that use predefined DVC anchors as placeholders for DVC to inject the plot
-values. You can create a custom template from scratch, or modify an existing one
-from `.dvc/plots/`.
-
-💡 Note that custom templates can be safely added to the template directory.
+File targets given to `dvc plots show` and `dvc plots diff` are treated as
+separate data series, each to be injected into a template file. There are two
+important fields that DVC adds to the plot data:
 
-All metrics files given to `dvc plots show` and `dvc plots diff` as input are
-combined together into a single data array for injection into a template file.
-There are two important fields that DVC adds to the plot data:
+- `step` - zero-based counter for the data rows/values. In many cases it
+  corresponds to a machine learning training epoch number.
 
-- `index` - zero-based counter for the data rows/values. In many cases it
-  corresponds to a machine learning training epoch or step number.
-
-- `rev` - Git commit hash, tag, or branch of the metrics file. This helps
-  distinguish between different versions when using the `dvc plots diff`
-  command.
+- `rev` - This field helps distinguish between data sourced from different
+  revisions, files or columns.
```

Comment:
Should we hide this in a `<details>` block? Doesn't seem like something 90% of users need to know.